### ISSUE TITLE
`project convert source` fails when passing in a package name

### ISSUE THREAD

**Issue by jimmysmith-ncino** [forcedotcom/cli#2063]
### Summary

When running `project convert source` with the `package-name` parameter passed in it will try to copy a directory using `fs.cpSync` without passing in the `recursive` flag. This can be found here:

https://github.com/salesforcecli/plugin-deploy-retrieve/blob/eb02e41c37eabb38e7b537c34967b5d37de9e059/src/commands/project/convert/source.ts#L146-L152

```typescript
    if (packageName && this.convertResult.packagePath) {
      // SDR will build an output path like /output/directory/packageName/package.xml
      // this was breaking from toolbelt, so to revert it we copy the directory up a level and delete the original
      fs.cpSync(this.convertResult.packagePath, outputDirectory);
      fs.rmSync(this.convertResult.packagePath, { recursive: true });
      this.convertResult.packagePath = outputDirectory;
    }
```

### Steps To Reproduce:

In an sfdx source project, run `sfdx project convert source --package-name "package"`

### Expected result

The source for the project is converted without errors.

### Actual result

```sh
Error (1): Path is a directory: cp returned EISDIR (/.../metadataPackage_x/package is a directory (not copied)) /.../metadataPackage_x/package
```

### System Information

```sh
% sfdx version --verbose --json
{
  "cliVersion": "sfdx-cli/7.196.7",
  "architecture": "darwin-arm64",
  "nodeVersion": "node-v16.15.1",
  "osVersion": "Darwin 22.4.0",
  "shell": "zsh",
  "rootPath": "...",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 2.1.8 (core)",
    "@oclif/plugin-commands 2.2.13 (core)",
    "@oclif/plugin-help 5.2.9 (core)",
    "@oclif/plugin-not-found 2.3.23 (core)",
    "@oclif/plugin-plugins 2.4.4 (core)",
    "@oclif/plugin-search 0.0.15 (core)",
    "@oclif/plugin-update 3.1.9 (core)",
    "@oclif/plugin-version 1.3.2 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.33 (core)",
    "@oclif/plugin-which 2.2.18 (core)",
    "@salesforce/lwc-dev-server 2.11.0 (user)",
    "apex 2.2.10 (core)",
    "auth 2.7.11 (core)",
    "community 2.2.9 (core)",
    "custom-metadata 2.1.11 (core)",
    "data 2.3.9 (core)",
    "deploy-retrieve 1.8.10 (core)",
    "info 2.6.4 (core)",
    "limits 2.3.12 (core)",
    "org 2.6.12 (core)",
    "packaging 1.16.3 (core)",
    "schema 2.3.6 (core)",
    "settings 1.4.3 (core)",
    "signups 1.4.11 (core)",
    "source 2.9.4 (core)",
    "telemetry 2.1.3 (core)",
    "templates 55.4.5 (core)",
    "trust 2.4.6 (core)",
    "user 2.3.7 (core)",
    "@salesforce/sfdx-plugin-lwc-test 1.0.1 (core)",
    "@salesforce/sfdx-scanner 3.5.1 (user)",
    "sfdx-cli 7.196.7 (core)"
  ]
}
```

**Comment by mhendy** [comment]
We're also experiencing this issue, downgrading to the previous stable version is a workaround. `sfdx update --version 7.194.1`

**Comment by WillieRuemmele** [comment]
Hi @jimmysmith-ncino and @mhendy thanks for reporting this, we've patched it in the plugin, and are releasing it as we speak

**Comment by WillieRuemmele** [comment]
This should be fixed in `7.196.8`, which is available on `npm` /tarballs now. the installers are on their way
